volatility3/framework/plugins/windows/driver_signing_enforcement.py

```diff
@@ -0,0 +1,125 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+import logging
+from typing import List
+
+from volatility3.framework import exceptions, renderers, interfaces, symbols
+from volatility3.framework.configuration import requirements
+from volatility3.framework.objects import utility
+from volatility3.plugins.windows import info, modules
+from volatility3.framework.symbols.windows import pdbutil
+
+vollog = logging.getLogger(__name__)
+
+"""
+A common technique by modern Windows rootkits is disabling Driver Signing Enforcement so that
+unsigned kernel rootkits can load. Rapid 7 documented many examples here:
+
+https://www.rapid7.com/blog/post/2021/12/13/driver-based-attacks-past-and-present/
+
+To disable DSE, malware will set the `g_CiOptions` global variable in the kernel to 0.
+
+This plugin locates this variable in memory and then reports its status and value.
+
+If this global variable is disabled on Windows 10 systems then it means the system is either in
+developer mode or that a rootkit purposely overwrote the value. Neither would be expected in production settings.
+"""
+class driver_signing_enforcement(interfaces.plugins.PluginInterface):
```

Comment:
Most of the classes in the plugin that currently exist use Pascal Case, and in fact, the class name also recommends that case. Of course, there are no clear guidelines or criteria for this in the current project. What do you think?

```suggestion
class DriverSigningEnforcement(interfaces.plugins.PluginInterface):
```